types/block.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	// MaxHeaderBytes is a maximum header size (including amino overhead).
-	MaxHeaderBytes int64 = 653
+	MaxHeaderBytes int64 = 632
```

Comment:
How do we verify this again?

Reply:
I think we have a test which calculates it and fails if the const is wrong

Reply:
https://github.com/tendermint/tendermint/blob/master/types/block_test.go#L239